packages/rancher-wins-upgrader/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,14 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,8 +1,8 @@
+ annotations:
+   catalog.cattle.io/os: windows
+-  catalog.rancher.io/certified: rancher
+-  catalog.rancher.io/namespace: cattle-wins-system
+-  catalog.rancher.io/release-name: rancher-wins-upgrader
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/namespace: cattle-wins-system
++  catalog.cattle.io/release-name: rancher-wins-upgrader
```

Comment:
Although persisting this change is being tracked in https://github.com/rancher/wins/pull/33, rancher/wins currently has no mechanism for releasing just a chart and not cutting a release for wins as a whole (tracked in https://github.com/rancher/wins/issues/34).

The next time we release wins, we can come back and remove this patch in favor of the new annotations on the chart in upstream.